haystack/pipelines/utils.py

```diff
@@ -250,31 +254,37 @@ def print_eval_report(
         n_wrong_examples=n_wrong_examples,
         document_scope=document_scope,
         answer_scope=answer_scope,
+        wrong_examples_fields=wrong_examples_fields,
+        max_chars=max_characters_per_wrong_examples_report,
     )
 
     print(f"{pipeline_overview}\n" f"{wrong_examples_report}")
 
 
-def _format_document_answer(document_or_answer: dict):
-    return "\n \t".join(f"{name}: {value}" for name, value in document_or_answer.items())
+def _format_document_answer(document_or_answer: dict, field_filter: List[str] = None):
+    if field_filter is None or len(field_filter) == 0:
+        field_filter = document_or_answer.keys()  # type: ignore
+    return "\n \t".join(f"{name}: {value}" for name, value in document_or_answer.items() if name in field_filter)  # type: ignore
```

Comment:
I suggest to pass `max_chars` here and cut the `value` by it.

Reply:
I'd rather not do that @tstadel because then the function does a few things rather than one. It's better to leave it as is, wdyt?

Reply:
I think it is fine for the method to be good at formatting one document/answer even if it means:
- filtering the fields
- cutting the values
- formatting the output per line

If we leave it like that I see two problem: 
- the name of the param `max_characters_per_field` does not really match its function
- the current "cutting" logic cuts all predictions, not only one. So this is implicitly replicating the same that `n_wrong_examples` does as it would probably only show the first prediction and nothing else.

So I would really go for cutting the individual field values. Another option would be to cut the individual predictions (after `_format_document_answer`) and not the field values. I would be ok with that too. But my gut feeling would be that cutting the field values is more powerful and gives users a better overview.